src/files.ts

```diff
@@ -129,7 +129,12 @@ export function parseFileIgnores(path: string): string[] {
       );
     }
   }
-  return parseIgnoreRulesToGlobs(rules, dirname);
+  try {
+    return parseIgnoreRulesToGlobs(rules, dirname);
+  } catch (err) {
+    console.error('Could not parse ignore rules to glob', { path });
+    throw new Error('Please make sure ignore file follows correct syntax');
```

Comment:
grammar nazi nit: every sentence should end with a dot (here and above). disclaimer: not a native speaker here 😄